.github/workflows/release.yml

```diff
@@ -75,33 +75,39 @@ jobs:
       - name: Setup PostgreSQL
         uses: ikalnytskyi/action-setup-postgres@v6
 
-      - id: build
+      - run: |
+          echo "HOST=$(rustc -vV | grep host: | awk '{print $2}')" >> $GITHUB_ENV
```

Comment:
nit: My personal preference would be to use outputs instead of environment variables because it's impossible to _unset_ environment variables in GitHub Actions, and because they may hypothetically clash with something else. :)

Reply:
Let me give it a try!